app/forms/candidate_interface/degrees/base_form.rb

```diff
@@ -0,0 +1,220 @@
+module CandidateInterface
+  class Degrees::BaseForm
```

Comment:
This is what is left of the [DegreeWizard](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/forms/candidate_interface/degree_wizard.rb). All the other forms inherit from it now, as there are some shared methods. It's main job is to collect all the data and coerce it into attributes that can be saved.